apps/store/src/features/priceCalculator/PurchaseFormV2/OfferPresenterV2/DeductibleSelectorV2/DeductibleSelectorV2.tsx

```diff
@@ -105,13 +105,13 @@ function DeductibleInfo() {
           {t('DEDUCTIBLE_SELECTOR_FOOTER_LINK')}
         </Button>
       </Dialog.Trigger>
-      <LargeDialog.Content closeOnPointerDownOutside={true}>
+      <LargeDialog.Content className={dialogContent} closeOnPointerDownOutside={true}>
```

Comment:
nit: we have 0 value available for sprinkles space values :)
```suggestion
      <LargeDialog.Content className={sprinkles({paddingBlock: 'none'})} closeOnPointerDownOutside={true}>
```

Reply:
Yes, but it wouldn't work because sprinkles styles would be overridden by LargeDialog.Content's.